### PR TITLE
fix: eth icon fallback

### PR DESCRIPTION
**Pull request by zhouhanseng**
@guanbinrui 

close https://github.com/DimensionDev/Maskbook/issues/1917

### CODE REVIEW

File: packages/maskbook/src/extension/options-page/DashboardComponents/TokenIcon.tsx [hunk 1]
```diff
@@ -6,18 +6,15 @@ import { CONSTANTS } from '../../../web3/constants'
...
-
-function resolveTokenIconURL(address: string) {
-    const checksummedAddress = formatChecksumAddress(address)
-    if (isSameAddress(checksummedAddress, getConstant(CONSTANTS, 'ETH_ADDRESS')))
-        return 'https://rawcdn.githack.com/trustwallet/assets/master/blockchains/ethereum/info/logo.png'
-    if (ICON_MAP[checksummedAddress]) return ICON_MAP[checksummedAddress]
-    return `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${checksummedAddress}/logo.png`
+    [formatChecksumAddress(
+        ETH_ADDRESS,
+    )]: 'https://rawcdn.githack.com/trustwallet/assets/master/blockchains/ethereum/info/logo.png',
```
Comment (septs) [inline packages/maskbook/src/extension/options-page/DashboardComponents/TokenIcon.tsx:17]: extract this link to constant variable.

Comment (septs) [reply packages/maskbook/src/extension/options-page/DashboardComponents/TokenIcon.tsx:17]: Unified use `raw.githubusercontent.com`.

```js
const TRUST_WALLET_ASSETS = 'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum';

`${TRUST_WALLET_ASSETS}/info/logo.png`
```

File: packages/maskbook/src/extension/options-page/DashboardComponents/TokenIcon.tsx [hunk 2]
```diff
@@ -40,11 +37,20 @@ export interface TokenIconProps extends withClasses<KeysInferFromUseStyles<typeo
...
     const tokenBlockie = useBlockie(props.address)
 
     return (
-        <Avatar className={classes.icon} src={resolveTokenIconURL(address)} {...props.AvatarProps}>
-            <Avatar className={classes.icon} src={tokenBlockie}>
+        <Avatar
+            className={classes.icon}
+            src={
+                ICON_MAP[checksummedAddress] ??
+                `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${checksummedAddress}/logo.png`
```
Comment (septs) [inline packages/maskbook/src/extension/options-page/DashboardComponents/TokenIcon.tsx:48]:
```
`${TRUST_WALLET_ASSETS}/assets/${checksummedAddress}/logo.png`
```

**Comment by Tedko** [comment]
should we fork the repo from trust wallet so we can make sure the source repo is safe (not been deleted)

@zhouhanseng @guanbinrui

**Comment by zhouhanseng** [comment]
I think we can fork, but still use the current one since its assets is updating everyday.

**Pull request merged**